text/0000-discord.md

```diff
@@ -0,0 +1,179 @@
+
+- a period of time when we use both chat platforms during the transition, put the equivalent Discord channel information in the Slack channel topic
+- a clear guide (with illustrations)
+- once all of the setup is complete, the Discord server invites can be distributed. 
+
+#### Initial Setup
+
+Because Discord has fine-grained controls, we will be able to implement categories for chats.
+
+We intend to have the "welcome" channel as the initial channel for everyone who joins the Discord server. This channel will be read-only and will list the rules for the Discord server.
+
+We also intend to have a "setup" channel. This channel will give you a complete guide of how to take advantage of the personalization, privacy and security, and notification controls in Discord. 
+
+**Verification Level**
+Initially, we will be implementing the "low" verification level, which means users will need to have a verified email on their Discord account. If this proves to be too easy of a target for spammers, we will implement a higher level of verification (levels include amount of time a user has to be a verified member of the server before they can post).
+
+**Explicit Content Filter**
+Since this is a public Discord server, we will be setting an explicit content filter- it will scan messages from all members without a role. Email-verified members will be given a community member role to start, and other roles may be added to users over time. 
+
+**Categories and Channels**
+Community members will then have the option of visiting the "setup" channel and learning more about fine-grained controls, such as:
+
+- notifications
+- muting a channel
+- muting a category
+
+Because our goal is transparency, all of the channels that exist will be visible in the channel list. A lock icon will display if the user does not have the role necessary to join that channel. (_FWIW, the alternative is to not display locked channels at all, which we felt would be less ideal- it is better to know that there are channels where private conversations are necessary and see what they are._)
+
+The following proposed initial category and channel list was chosen based on the current channel needs and evaluation of the channels with the most members on Slack. _Additional channels may be requested in the Admin/community-feedback channel._
+
+**Category/Channel List:**
+- (No Category)
+  - welcome (community guidelines are posted here) <--readonly & the server invite puts users in this channel first. 
+  - setup-profile (how to setup your profile) <--readonly
+- Admin
+  - community-feedback (questions, comments, concerns, requests)
+  - security
+  - steering-committee 🔒 (locked to role “steering-committee”)
+  - news & announcements
+  - ember-jobs
+  - bots
+- Core Teams
+  - ember-js 🔒 (locked to role “core-js”)
+  - ember-data 🔒 (locked to role “core-data”)
+  - ember-cli 🔒 (locked to role “core-cli”)
+  - ember-learning 🔒 (locked to role “core-learning”)
+- Working on Ember
```

Comment:
Need testing channel both under 'Working on Ember' and 'Using Ember'

Reply:
Also a 'Release' channel under 'Working on Ember'

Reply:
@kategengler should we use the dev- prefix that is currently being used in Slack?

Reply:
I think the category headings are enough, but maybe we see how it feels? I was just pointing out the missing channels that are heavily used.

Reply:
I problem with that is it kind of breaks the autocomplete, which would be useful for things like "you should ask that question on #ember-cli". I can see autocompleting to the wrong channel becoming a pretty common problem:

<img width="690" alt="j" src="https://user-images.githubusercontent.com/55829/43224295-b47ea372-900a-11e8-9168-1d6b98d5bffd.png">